lua/ttt2/extensions/draw.lua

```diff
@@ -678,6 +678,35 @@ function draw.GetTextSize(text, font, scale)
     return w * scale, h * scale
 end
 
+---
+-- Creates a text that is limited to the provided length. Adds a limiting chat (e.g. '...') after the
+-- text if so desired.
```

Comment:
You should clarify that the limiting char is not counted in to the length.

also typo in "char"